third-party/proxygen/CMakeLists.txt

```diff
@@ -2,6 +2,38 @@ set(PROXYGEN_LIB_DIR "${CMAKE_CURRENT_SOURCE_DIR}/src/proxygen/lib")
 set(PROXYGEN_HTTPSERVER_DIR "${CMAKE_CURRENT_SOURCE_DIR}/src/proxygen/httpserver")
 set(PROXYGEN_EXTERNAL_DIR "${CMAKE_CURRENT_SOURCE_DIR}/src/proxygen/external")
 
+# ported over from src/CMakeLists.txt
+set(PROXYGEN_FBCODE_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/src)
+set(PROXYGEN_GENERATED_ROOT ${CMAKE_CURRENT_BINARY_DIR}/generated)
+file(MAKE_DIRECTORY ${PROXYGEN_GENERATED_ROOT})
+
+find_program(PROXYGEN_PYTHON python)
```

Comment:
If there's issues on some builds, likely to need to add python3 to build-depends in debian control files, and try `python3` first, falling back to `python` as the executable